### PR TITLE
#8 flask initial set up

### LINKED ISSUE #8: Python Flask Infra Setup - Getting the backend server running

### PR DESCRIPTION
**Pull request by lillianjiang**

### CODE REVIEW

File: main.py [hunk 1]
```diff
@@ -0,0 +1,11 @@
+from flask import Flask
+
+app = Flask(__name__)
```
Comment (g1isgone) [inline main.py:3]: Let's put a separate commit in the README.md on how to run the Flask server @elainoz @kvg-png  @ntresvalles1  @naylynn-TR  this is where the sql alchemy code will come; 

You will have to instantiate the sql flask db here 

`app.config['SQLALCHEMY_DATABASE_URI'] = 'sqlite:////tmp/test.db'
db = SQLAlchemy(app)`

File: venv/bin/activate [hunk 2]
```diff
@@ -0,0 +1,76 @@
+# This file must be used with "source bin/activate" *from bash*
```
Comment (g1isgone) [inline venv/bin/activate:1]: is this venv for virtual environments to manage package dependencies independent of the os?

Comment (lillianjiang) [reply venv/bin/activate:1]: It depends on operating system. For Mac OS, `venv/bin/activate` or `source venv/bin/activate`. For Windows, `venv\Scripts\activate`.
